src/test/operators/print_test.cpp

```diff
@@ -0,0 +1,120 @@
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "../base_test.hpp"
+#include "gtest/gtest.h"
+
+#include "operators/get_table.hpp"
+#include "operators/print.hpp"
+#include "storage/storage_manager.hpp"
+#include "storage/table.hpp"
+
+namespace opossum {
+
+// class OperatorsPrintTest : public BaseTest {
```

Comment:
They are now. Btw. the print tests aren't activated in your sprint3 branch as well. I checked because my compiler complained about line 33 in `print_test.cpp`. But it's an easy fix. I activated them now